src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php

```diff
@@ -24,13 +25,35 @@
  */
 class HttpFoundationExtension extends AbstractExtension
 {
-    private $requestStack;
-    private $requestContext;
+    private $urlHelper;
 
-    public function __construct(RequestStack $requestStack, RequestContext $requestContext = null)
+    /**
+     * @param UrlHelper $urlHelper
+     */
+    public function __construct($urlHelper)
```

Comment:
You should not inject your helper here, create a new Runtime instead.

Reply:
As discussed, the gain is too small, so I'll keep it the way it is.